examples/docs_snippets/docs_snippets/integrations/tableau/customize-tableau-asset-defs.py

```diff
@@ -21,8 +22,20 @@
 # Tableau assets, such as the owners or asset key
 class MyCustomTableauTranslator(DagsterTableauTranslator):
     def get_sheet_spec(self, data: TableauContentData) -> dg.AssetSpec:
-        # We add a custom team owner tag to all sheets
-        return super().get_sheet_spec(data)._replace(owners=["team:my_team"])
+        # We create the default asset spec for a given sheet using super()
+        default_spec = super().get_sheet_spec(data)
+        # We customize the team owner tag only for sheets
+        return replace_attributes(default_spec, owners=["team:my_team"])
+
+    def get_asset_spec(self, data: TableauContentData) -> dg.AssetSpec:
+        # We create the default asset spec using super()
+        default_spec = super().get_asset_spec(data)
+        # We customize the metadata and asset key prefix for all assets, including sheets
+        return replace_attributes(
```

Comment:
Do we want to use `default_spec.replace_attributes`, once that merges?

Reply:
I think we can reduce the verbosity of this a bit once that merges - no need to set `default_spec` & can just do `return super().get_asset_spec(data).replace_attributes(....)`

Reply:
> Do we want to use default_spec.replace_attributes, once that merges?

Definitely - it's mentioned in the PR description, but I could have added a comment here to make it clearer.

> I think we can reduce the verbosity of this a bit once that merges - no need to set default_spec & can just do return super().get_asset_spec(data).replace_attributes(....)

I think it works if we keep the comment and note after the code snippet.

Reply:
`I think we can reduce the verbosity of this a bit once that merges - no need to set default_spec & can just do return super().get_asset_spec(data).replace_attributes(....)`

Actually, we reuse `default_spec` in the parameters of `replace_attributes`, so we should keep `super().get_asset_spec(data)` and `replace_attributes(....)` to avoid calling `super()` again to access the asset key. Otherwise we get this:

```python
return super().get_asset_spec(data).replace_attributes(
    key=super().get_asset_spec(data).key.with_prefix("my_prefix")
)
```

Also, having `default_spec = super().get_asset_spec(data)` puts the emphasis on creating the default spec first.